### DISCUSSION TITLE
无法登陆评论

### DISCUSSION THREAD

**Discussion by jokerlzl** [walinejs/waline#397] [Q&A] [answered]
有人帮帮我吗

**Comment by lizheming** [comment]
提供一下你的地址我们看一下

**Reply by jokerlzl** [reply]
https://jokerblog.vercel.app/

**Comment by Mister-Hope** [comment, marked as answer]
I am afraid you haven't configure the database env variables and not yet registered to become administritor.

Please read the [docs](https://waline.js.org/en/guide/get-started.html) carefully.

---

恐怕您还没有配置数据库环境变量，也没有注册成为管理员。

请仔细阅读 [文档](https://waline.js.org/guide/get-started.html)。
